.travis.yml

```diff
@@ -37,7 +37,10 @@ before_script:
   - echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
 
 script:
-  - vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+  - vendor/bin/phpunit -c phpunit.xml
```

Comment:
Ok, the PHPUnit does not need ```-c phpunit.xml``` and it can automatically detect the phpunit.xml file.
I just use the command: ```vendor/bin/phpunit```